bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionGuardException.cs

```diff
@@ -42,7 +43,16 @@ public ElectionGuardException(string message, Status code) : base(message)
         /// <param name="message">message to be used in the exception handling</param>
         /// <param name="inner">exception to wrap inside the ElectionGuardException</param>
         /// <returns>New ElectionGuardException object</returns>
-        public ElectionGuardException(string message, Exception inner) : base(message, inner) { }
+        public ElectionGuardException(string message, Exception inner) : base(message, inner) { }
+
+        public static void ThrowIfNull(object argument, string message = null)

```

Comment:
Added this to be like the ArgumentException.ThrowIfNullOrempty methods that the newer version of .net have built in.